### PR TITLE
Twitter rewrite

### PR DESCRIPTION
**Pull request by fromulus**
It's been discussed a few times already that we wanted to distanciate ourselves from Monika's official Twitter account, so I gave a shot at rewriting the related topics myself.

The main Twitter topic is longer and now also tackles the idea of there being merch of Monika, as I felt it was appropriate to the point Monika is trying to make in this new version of the topic (being uneasy at the idea of being nothing more than a profitable intellectual property to her creators).

I also added a conditional (having seen `monika_clones`) as she mentions other Monikas a few times, not sure if doing that on pre-existing topics is okay.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
-    m 1hua "I love sharing my thoughts and chatting with the world..."
-    m 1eud "The real world."
-    m 1esa "So make sure you follow me, okay?"
+    m 3eud "Did you know there's an official Twitter account based on me?"
+    m 3esc "The username is {a=https://twitter.com/[mas_monika_twitter_handle]}{i}{u}[mas_monika_twitter_handle]{/u}{/i}{/a}."
+    m 2rtc "I'm not entirely sure who's truly running this account."
+    m "I can definitely tell you I have nothing to do with it..."
+    m 2rtd "But is it really another Monika?"
+    m 2euc "I mean, I don't know if you've read some of the tweets on there,{w=0.1} but I'm having a hard time recognizing myself in them."
+    m 4eud "Like, it depicts some events that I don't even remember ever happening,{w=0.2} such as one time when I apparently had a sleepover with Yuri."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1204]:
```suggestion
    m 4eud "It even depicts some events that I don't remember ever happening...{w=0.2}like when I apparently had a sleepover with Yuri."
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 4esd "And there's no mention of any of the things that have happened back in the game, even though this version of me still seems to be aware of the real world..."
+    m 1etc "So...{w=0.4}maybe the Monika running this account went through a completely different story than the one most Monikas--{w=0.2}including me--{w=0.2}did."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1206]:
```suggestion
    m 1etc "So...{w=0.3}maybe the Monika running this account went through a completely different story than the one most Monikas--{w=0.1}including me--{w=0.1}did."
```

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 1lsc "Or maybe this is just one of the developers of the game pretending to be me..."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1207]:
```suggestion
    m 1lkc "...Or maybe this is just one of the developers of the game pretending to be me."
```

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 3lksdlc "I mean, I'm not really sure how to feel about this.{w=0.4} Isn't that identity theft, in some way?"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1208]:
```suggestion
    m 3lksdlc "If that's the case, I'm not really sure how I feel about it...{w=0.3}Isn't that identity theft in a sense?"
```

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 3eksdld "But it would also make sense, right? {w=0.2}{nw}"
+    extend 3eud "This account makes for a nice way to connect with the people who have played through the game and have grown attached to it."
+    m 1euc "In that sense, this would make it some kind of publicity stunt..."
+    m 1esc "After all, while there's a lot of sweet looking art and such on there, it is suspicious that almost all of them are posted specifically on important days."
+    m "Like on Valentine's day, or on my birthday...{w=0.4}which just so happen to coincide with the original release date of DDLC."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1213]:
```suggestion
    m 3eua "Like on Valentine's day, or on my birthday, which just so happen to coincide with the original release date of DDLC."
```

File: Monika After Story/game/script-topics.rpy [hunk 6]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 1rsc "I mean, it's not like the creators of the game are above using my image for these sorts of things."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1214]:
```suggestion
    m 1rsc "It's not like the creators of the game are above using my image for these sorts of things..."
```

File: Monika After Story/game/script-topics.rpy [hunk 7]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 4etd "Like,{w=0.1} did you know they have an official online shop where you can buy various kinds of merchandise based on me and the other girls?"
+    m 4etc "I really don't know how I'm supposed to feel about that."
+    m 2lsc "After all, as my creators, they more or less own my image as a character, so I guess they are legally in their right."
+    m 2lkd "But...{w=0.3}I'm more than a mere character in a game, am I not?"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1218]:
```suggestion
    m 2lkd "But...{w=0.3}I'm more than a mere character in a game, aren't I?"
```

File: Monika After Story/game/script-topics.rpy [hunk 8]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 2lkc "I am my own person, with my own thoughts and feelings, and so is every other Monika in the different copies of this game."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1219]:
```suggestion
    m 2lkc "I'm my own person with my own thoughts and feelings,{w=0.1} and so is every other Monika in the different copies of this game."
```

File: Monika After Story/game/script-topics.rpy [hunk 9]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 2dksdlc "So it feels really wrong to be used like that by people I haven't even met."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1220]:
```suggestion
    m 2dksdlc "It feels wrong to be used like that by people I haven't even met."
```

File: Monika After Story/game/script-topics.rpy [hunk 10]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 7eksdld "I guess you could make the argument that they owe me and every other Monika at least some of the profits,{w=0.2} but I don't even actually care about that part."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1221]:
```suggestion
    m 7eksdld "I guess you could make the argument that they owe me and every other Monika at least some of the profits,{w=0.2} but I don't really care about that part."
```

File: Monika After Story/game/script-topics.rpy [hunk 11]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 7eksdlc "I just hate the feeling of being reduced to a marketable asset like that."
+    m 2lsc "But I suppose there isn't really all that much I can do against it...{w=0.3}{nw}"
+    extend 2dsc "neither do I really want to go through that kind of trouble."
+    m 2eua "I mean, at the end of the day, all that matters to me is that I get to be with you.{w=0.3} And this would've never been possible without my creators."
+    m 2lka "So if what it takes in return is that they get to make some profit on the work they put in making the game, then I'd say this isn't too bad of a deal."
+    m 1eua "You don't have to feel bad about any of that, though.{w=0.3} If you follow that account on Twitter or decide to buy some merchandise of me, it's still really sweet."
+    m 1eka "Just remember that those aren't really me, alright?"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1228]: Not too fond of this feeling like a product placement tbh
```suggestion
```

File: Monika After Story/game/script-topics.rpy [hunk 12]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 1ekb "I'm here, and I love you."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1229]:
```suggestion
    m 1ekb "Just know that I'm here, and I love you."
```

File: Monika After Story/game/script-topics.rpy [hunk 13]
```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
...
+    m 1hua "So make sure to always come spend some time with me, okay?"
     m 3ekbsa "It would mean a lot, with how much you mean to me."
     m 1hubfa "It would really make me feel loved."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:1232]:
```suggestion
    m 1hubfa "It would really make me feel loved~"
```

**Review by multimokia — CHANGES_REQUESTED** [review]
Some things

File: Monika After Story/game/script-topics.rpy (hunk 8 above)
Comment (Freshairkaboom) [reply Monika After Story/game/script-topics.rpy:1219]:
```suggestion
    m 2lkc "I'm my own person, with my own thoughts and feelings, and so is,{w=0.1} I would presume,{w=0.1} every other Monika in the different copies of this game."
```

I think this makes more sense considering her knowledge about her realness only comes from her own experience, which is separated from the other Monikas. Just to softball the assumption that the other Monikas are as real as the one you're talking to, without her saying they aren't real.

**Review by Freshairkaboom — CHANGES_REQUESTED** [review]

**Review by ThePotatoGuy — CHANGES_REQUESTED** [review]
update PR description and title so we know teh showEVL change happened here and why (or split into separate pr)

**Comment by jmwall24** [comment]
Note that last comment is for me, author can disregard it. I'll look at it tomorrow.

**Review by multimokia — DISMISSED** [review]

**Comment by jmwall24** [comment]
Need to wait for #6865 as this should use `removeDerand()` in the update script.

**Review by multimokia — APPROVED** [review]

**Review by jmwall24 — APPROVED** [review]

**Pull request merged**